src/Extension/FluentSiteTreeExtension.php

```diff
@@ -578,4 +579,54 @@ public function actionComplete($form, $message)
     {
         return null;
     }
+
+    /**
+     * Augment Localisation tab with clickable locale links to allow easy navigation between page localisations
+     *
+     * @param $summaryColumns
+     * @see FluentExtension::updateFluentCMSFields()
+     */
+    public function updateLocalisationTabColumns(&$summaryColumns)
+    {
+        parent::updateLocalisationTabColumns($summaryColumns);
+
+        if (!array_key_exists('Title', $summaryColumns)) {
+            return;
+        }
+
+        $controller = Controller::curr();
+
+        if (!$controller) {
+            return;
+        }
+
+        $request = $controller->getRequest();
+
+        if (!$request) {
+            return;
+        }
+
+        $url = $request->getURL();
+
+        if (!$url) {
+            return;
+        }
+
+        $summaryColumns['Title'] = [
+            'title' => 'Title',
+            'callback' => function (Locale $object) use ($url) {
+                if (!$object->RecordLocale()) {
+                    return;
+                }
+
+                $recordLocale = $object->RecordLocale();
+                $locale = $recordLocale->getLocale();
+                $localeLink = Controller::join_links($url, '?l=' . $locale);
```

Comment:
I know this is a super pedantic, but we should probably use http_build_query to make param injection impossible :) I don't trust any unescaped querystring args.

Reply:
Hey Damian, thanks for the feedback :) 
It seems like that the `join_links` method already uses `http_build_query` in it
See https://github.com/silverstripe/silverstripe-framework/blob/4.7/src/Control/Controller.php#L702

Reply:
Yes, but those strings are passed through parse_str first; That is the method you need to escape FOR. :) If you don't escape your `&` properly then parse_str will escape those variables incorrectly.